Solutions/Corvus.Monitoring.AspnetCore.Mvc.Demo/Controllers/HomeController.cs

```diff
@@ -0,0 +1,36 @@
+﻿// <copyright file="HomeController.cs" company="Endjin Limited">
+// Copyright (c) Endjin Limited. All rights reserved.
+// </copyright>
+
+namespace Corvus.Monitoring.AspnetCore.Mvc.Demo.Controllers
+{
+    using Corvus.Monitoring.AspnetCore.Mvc.Demo.Models;
+    using Microsoft.AspNetCore.Mvc;
+
+    /// <summary>
+    /// Controller for the home page.
+    /// </summary>
+    public class HomeController : BaseController
+    {
+        /// <summary>
+        /// Action method for the home page.
+        /// </summary>
+        /// <returns>A view result for the home page.</returns>
+        [HttpGet]
+        public async Task<IActionResult> Index()
+        {
+            HomeViewModel model = new();
+
+            // Since this.CurrentOperation is nullable, we use the null-conditional operator to avoid compiler warnings/runtime exceptions
```

Comment:
> to avoid compiler warnings/runtime exceptions

Is avoiding runtime exceptions really what we want? Isn't that a "this shouldn't happen" scenario, so we'd want an exception if we've done something that has stopped this from working?

Reply:
Badly explained, I think. It's one of those scenarios where it could be null if you hadn't applied the action filter, but we know we have so it won't be null. I think the changes discussed in the comments above (https://github.com/corvus-dotnet/Corvus.Monitoring/pull/199#discussion_r1291097304) will remove the need for this.

Reply:
No longer relevant with the latest changes.